internal/criocli/criocli_test.go

```diff
@@ -48,8 +48,8 @@ var _ = t.Describe("CLI", func() {
 
 	It("should return a copy of the slice", func() {
 		// Given
-		slice.Set("value1")
-		slice.Set("value2")
+		Expect(slice.Set("value1")).To(BeNil())
```

Comment:
I think this change snuck in, can we have indication it's added in the pr or commit message?

Reply:
Yep, I'll add a note to fix the linter.